nupic/data/stats_v2.py

```diff
@@ -265,7 +265,7 @@ def generateStats(filename, maxSamples = None,):
                            'bool':     BoolStatsCollector,
                            }
 
-  filename = findDataset(filename)
+  filename = resource_filename("examples.prediction.data", filename)
```

Comment:
the first arg here is a string path, that may be why you don't need `__init__.py` files in `example` and `prediction` dirs. That is nice in this case but I prefer to pass an imported module as the first argument. The import mechanism is easier to debug and makes the dependency graph explicit. Do you think it is better to use the string version?